Makefile

```diff
@@ -84,6 +86,7 @@ KIND_VERSION ?= 0.8.1
 KUBERNETES_VERSION ?= 1.18.2
 BATS_VERSION ?= 1.2.1
 TRIVY_VERSION ?= 0.14.0
+PROTOC_VERSION ?= 3.13.0
```

Comment:
Why not use the latest version `v3.15.2`?

Reply:
I was bumbling my way through and wanted to not change too much at once. This looks like a simple update (no changes to generated code) so I'll bump to 3.15.2